helm/templates/pod_disruption_budget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
```

Comment:
Like @aianus said, let's wrap this in a conditional with a flag defined in values yaml